test/Frontend/Desugar.hs

```diff
@@ -1,59 +1,101 @@
 module Frontend.Desugar where
 
-import Data.Attoparsec.ByteString
-import qualified Data.Attoparsec.ByteString.Char8 as Char8
+import Data.Attoparsec.ByteString (parse)
+import Frontend.Parser (shouldParseAs)
 import qualified Juvix.Frontend.Parser as Parser
-import Juvix.FrontendDesugar
-import Juvix.Library hiding (show)
+import Juvix.Frontend.Types (TopLevel)
+import Juvix.Frontend.Types.Base
+  ( Cond' (C'),
+    CondLogic'
+      ( CondExpression',
+        annCondExpression,
+        condLogicBody,
+        condLogicPred
+      ),
+    Constant' (Number'),
+    Expression' (Constant', Infix', Name'),
+    Function' (Func'),
+    FunctionLike'
+      ( Like',
+        annLike,
+        functionLikeArgs,
+        functionLikeBody,
+        functionLikedName
+      ),
+    GuardBody' (Guard'),
+    Infix' (Inf', annInf, infixLeft, infixOp, infixRight),
+    Numb' (Integer''),
+    TopLevel' (Function'),
+  )
+import Juvix.Library
+  ( Alternative (many),
+    ByteString,
+    NonEmpty ((:|)),
+    Symbol (Sym),
+  )
```

Comment:
I'd much rather keep the qualified imports and deal with the verbosity by using |>'s